models/image.py

```diff
@@ -853,11 +647,14 @@ def from_images(cls, images):
         if len(images) < 1:
             raise ValueError("Must provide at least one image to combine.")
 
+        # sort images by mjd:
+        images = sorted(images, key=lambda x: x.mjd)
+
         output = Image(nofile=True)
 
         # for each attribute, check that all the images have the same value
         for att in ['section_id', 'instrument', 'telescope', 'type', 'filter', 'project', 'target']:
```

Comment:
I wouldn't require the project and target to be the same.  Target is probably going to be parsed from the FITS header, and there are reasons why it might vary.  Sometimes people type differently (spaces, no spaces, capitalization), and sometimes they just get it wrong.

It's also entirely possible that we will want to coadd images from different projects in order to make a deep reference.  I've certainly done this with the NOIRLab archive.

Finally, we may well need to coadd images with different section_ids.  Even with LS4, the same field will show up on different chips sometimes, because the quartered filter on the focal plane has two i band quadrants, so sometimes a given field will be in the top half of the detector, sometimes in the bottom half.  But, in general, surveys will not always go back to exactly the same fields, so you need to build references as mosaics of mulitple other images.

Reply:
The project really doesn't have to be the same, that's easy to remove. 
The problem is that if you are on a grid of fields, then `target` will tell you the field ID and `section_id` will tell you where you are on that grid. This is the only thing we have right now that prevents coadding/subtracting the wrong sky coordinates. 

I suggest that we make a separate PR where we add the functionality of checking if images are overlapping or not, before combining them, and then use that instead of these restrictions. 

in the meantime I will try to see if we actually need to have values for the `section_id` and `project` and `target` in the coadded / subtracted image, and how critical is it if we just pick the values from one image (risking that it will be different from the others).

Reply:
I think this comment still stands; I really won't put target here, as that will come back to bite us.  (Consider sky flats, if nothing else-- that's something that doesn't match RA/Dec either!)  We've also discussed earlier about how section_id will need to be different in combinations we make.  For the most general consideration of image combaintion, we should just be checking instrument, telescope, filter, and perhaps type.  In this function, only those general considerations should be checked.  More specific considerations will come from the code that implements the actual combination.

As for setting the attribute in the combined image, if the corresponding attributes aren't the same in all the input images, they should be set to some placeholder value that indicates it comes from something heterogeneous.  Or, it might be different for different combinations.

Reply:
I added this as Issue #179